package.json

```diff
@@ -3,22 +3,50 @@
   "version": "0.2.1",
   "description": "JavaScript style rules for MITODL",
   "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
   "keywords": [
     "eslint",
     "mitodl",
     "mit"
   ],
+  "scripts": {
+    "test": "eslint --report-unused-disable-directives ./tests"
```

Comment:
The tests are very basic: just asserting that lint errors are thrown where they should be.

This might seem weird... But given how easy it is to completely disable linting by making a grep mistake, it seems worthwhile.